content/docs/introduction/billing.md

```diff
@@ -0,0 +1,188 @@
+- Neon's _Configurable auto-suspend compute_ feature, which allows you to configure the timeout period for the  _Auto-suspend compute_ feature (_coming soon_).
+- Neon's _Always-on_ compute feature, which keeps an endpoint active indefinitely to avoid the few seconds of connection latency when waking an idle compute (_coming soon_).
+- Neon's _autoscaling_ feature, which allows you to set a minimum and maximum number of CUs for each compute endpoint. The number of active CUs scale up and down based on workload (_coming soon_).
+
+
+The cost calculation for _Compute time_ is as follows:
+
+```text
+compute units * active time (hours) * price per hour
+```
+
+### Project storage
+
+The _Project storage_ metric counts the amount of data stored in all of your Neon projects. Project storage includes:
+
+- **Logical data size**
+
+  The size of all databases in your Neon projects. You can think of this as a _snapshot_ of your data at a point in time.
+
+- **Retained Write-Ahead Log (WAL) records**
+
+  Neon retains Write-Ahead Log (WAL) records to support _point-in-time restore_ and _database branching_.
+  - _Point-in-time restore_ is the ability to restore data to a past point in time. Neon stores seven days of data history in the form of WAL records for a Neon project. WAL records that fall out of this seven-day window are evicted from storage and no longer counted toward project storage.
+  - A _database branch_ is a snapshot of your data (including _data history_) at the point of branch creation combined with WAL records that capture data changes from that point forward.
+    When a branch is first created, it adds no storage. No data changes have been introduced yet, and the branch's snapshot data still exists in the parent branch's _data history_, which means that it shares the snapshot data in common with the parent branch. A branch only begins adding to storage when data changes are introduced or when the branch snapshot falls out of the parent branch's _data history_, in which case the branch's snapshot data is no longer shared in common. In other words, branches add storage when you modify data and allow the branch to age out of the parent branch's _data history_.
+
+    Database branches can also share a _data history_. For example, two branches created from the same parent at or around the same time will share a _data history_, which avoids additional storage. The same is true for a branch created from another branch. Wherever possible, Neon minimizes storage through shared data history. Additionally, to keep storage to a minimum, Neon will take a new branch snapshot if the amount data changes stored as WAL grows to the point that a new snapshot would consume less storage.
+
+The cost calculation for _Project storage_ is as follows:
+
+```text
+project storage (GiB) * (seconds stored / 3600) * price per hour
```

Comment:
In the UI we plan to show both the current value of the Project storage and also the accrued GiBh since the beginning of the billing period. tentatively we suggest to call the latter 'Project storage (cum.)', but i'm very much looking for the alternative proposals